src/tool/cppwinrt/strings/base_error.h

```diff
@@ -110,6 +107,24 @@ WINRT_EXPORT namespace winrt
                     WINRT_VERIFY_(impl::error_ok, info2->CapturePropagationContext(nullptr));
                 }
             }
+            else
+            {
+                impl::bstr_handle legacy;
+
+                if (info)
+                {
+                    info->GetDescription(legacy.put());
+                }
+
+                hstring message;
```

Comment:
Consider if you get want to append GetGuid / GetSource to the message.  Not sure how often they are used.

Reply:
Rarely in my experience, and then I'd have to worry about how to format it. 😨